CONTRIBUTING.md

```diff
@@ -28,6 +28,7 @@ below:
 
 <!-- start-shortlog -->
  - Oliver Sanders
+ - Bruno P. Kinoshita
```

Comment:
We now automatically check the names in the CLA against the mailmap so they need to be in the same format.

I.E. "Bruno Kinoshita" rather than "Bruno P. Kinoshita"